elife/aws-cli.sls

```diff
@@ -1,4 +1,3 @@
 aws-cli:
     cmd.run:
-        - name: pip install --upgrade 'awscli~=1.16'
-
+        - name: python3 -m pip install --upgrade 'awscli~=1.16'
```

Comment:
good to know another way to invoke pip from Python 3. I've been using `pip3`so far, which I think is equivalent

Reply:
yup.

I prefer the longish `python3 -m foo bar` approach because then you only need to figure out which python3 is being used. `pip` and `pip3` within a virtualenv are identical if it's a python3 venv, `pip` and `pip3` outside a virtualenv are not.